src/components/pages/notifier/sell/NotifierSellPage.tsx

```diff
@@ -29,6 +33,25 @@ const NotifierSellPage: FC = () => {
   const [isCheckingWhitelist, setIsCheckingWhitelist] = useState(true)
   const [isWhitelistedProvider, setIsWhitelistedProvider] = useState(false)
   const accountStr = account as string // wrapped with login card
+  const [isLoadingOffer, setIsLoadingOffer] = useState(false)
+  const [hasOffer, setHasOffer] = useState(false)
+
+  useEffect(() => {
+    const getOwnOffers = async (): Promise<void> => {
+      setIsLoadingOffer(true)
+
+      const offersService = apis['notifier/v0/offers'] as NotifierOffersService
+      offersService.connect(reportError)
+
+      const currentOwnOffers = await offersService.fetch({
+        provider: account,
+      })
+      setHasOffer(Boolean(currentOwnOffers[0]))
```

Comment:
Might be safer for runtime to do:
```suggestion
      setHasOffer(Boolean(currentOwnOffers?.length))
```